.github/workflows/config/spelling_allowlist.txt

```diff
@@ -214,19 +224,27 @@ lossy
 lvalue
 makefiles
 merchantability
+mgpu
```

Comment:
I don't think this is a word and therefore should probably not be added to the allowlist.